helper/include/odph_list_internal.h

```diff
@@ -18,30 +18,22 @@
 extern "C" {
 #endif
 
-/** List object */
+/** @cond __INTERNAL_DOCUMENTATION__ */
```

Comment:
To be more consistent with the rest of the implementation `_ODP_HIDE_FROM_DOXYGEN_` could be used here.